sdk/schemaregistry/azure-schemaregistry-avroserializer/setup.py

```diff
@@ -65,7 +65,7 @@
     zip_safe=False,
     packages=find_packages(exclude=exclude_packages),
     install_requires=[
-        'azure-schemaregistry==1.0.0b1',
+        'azure-schemaregistry==1.0.0b2',
```

Comment:
version.py also needs update to 1.0.0b2?

Reply:
weirdly enough, that's already at 1.0.0b2? https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/schemaregistry/azure-schemaregistry-avroserializer/azure/schemaregistry/serializer/avroserializer/_version.py